zhaquirks/centralite/3460L.py

```diff
@@ -30,11 +30,8 @@ class CustomPowerConfigurationCluster(PowerConfigurationCluster):
     """Custom PowerConfigurationCluster."""
 
     cluster_id = PowerConfigurationCluster.cluster_id
-    MIN_VOLTS = 21
-    MAX_VOLTS = 31
-
-    def _update_attribute(self, attrid, value):
-        super()._update_attribute(attrid, value)
+    MIN_VOLTS = 2.1
+    MAX_VOLTS = 3.1
```

Comment:
Just override the min

Reply:
Eh, let’s make it 3.0

Reply:
Im using the previous comment to remove the max.

Reply:
Fixed.